08_calculator/solution/calculator-solution.js

```diff
@@ -10,11 +10,13 @@ const sum = function (array) {
   return array.reduce((total, current) => total + current, 0);
 };
 
-const multiply = function (array) {
-  return array.length
-    ? array.reduce((accumulator, nextItem) => accumulator * nextItem)
-    : 0;
-};
+const multiply = function(...args){
+  let sum = 1
+ for (let i = 0; i < args.length; i++) {
```

Comment:
this is really nitpicky.. but can you fix the indentation on this line please.

Reply:
I've made both changes through the 'edit' function here now. Hope that works, otherwise let me know if I need to open a new PR.